sequencer.c

```diff
@@ -1033,7 +1035,13 @@ static int run_git_commit(const char *defmsg,
 		strvec_push(&cmd.args, "--cleanup=strip");
 	if ((flags & VERBATIM_MSG))
 		strvec_push(&cmd.args, "--cleanup=verbatim");
-	if ((flags & EDIT_MSG))
+	if ((flags & EDIT_MSG) && (flags & INLINE_MSG)) {
+		memcpy(message, opts->subject, opts->subject_len);
+		memcpy(message + opts->subject_len, opts->content,
+				opts->content_len);
+		strvec_push(&cmd.args, "-m");
+		strvec_push(&cmd.args, message);
```

Comment:
This can easily run into command-line limits. That would not be an issue if we only ever pass the `cmd.args` to an internal function, but it _is_ a problem if we pass this to an external process. Have you looked into that?